src/sklearn_evaluation/plot/calibration.py

```diff
@@ -282,11 +282,6 @@ def __add__(self, another):
             cmaps=[self.cmap, another.cmap],
         ).plot()
 
-    def __sub__(self, another):
```

Comment:
you're on the right track. remove everything that looks like this

Reply:
Got it! I have a project deadline on Tuesday so probably need to work on that first. I'll try to have this fixed by Monday night.

Reply:
sure

Reply:
@edublancas  Hey! I'm wrapping this up with the generic sub method today. A quick question. For the sub method, since we are not defining anything in the abstract class constructor, what value should I use in the return self.value - another.value? Does this mean I should also define something in the constructor? thanks!

Reply:
> what value should I use in the return self.value - another.value? 

I'm not following this.

the idea is to remove any `__sub__` method that raises a NotImplementedError, then implement a `__sub__` method that raises the NotImplementedError in the abstract class. where is the ` self.value - another.value` part?

Reply:
I thought by a generic __sub__ method, we still want a self - others definition in the ab class. Thanks for clarifying, the NotImplementedError part was already implemented last week.

Reply:
no problem! so is this ready for review or are there any pending changes?

I see there are some conflicts so you'll need to rebase

Reply:
got it, rebasing it now

Reply:
yes I think it is ready for review after rebase